CHANGES.md

```diff
@@ -24,6 +24,9 @@ Tactics
   Simplex-based proof engine. In case of regression, 'Unset Simplex'
   to get the venerable Fourier-based engine.
 
+Vernacular Commands
+
+- Removed the `Implicit Tactic` family of commands.
```

Comment:
Maybe add the "deprecated" qualifier here so that people can see at a glance that we are following the normal process of deprecating first, then removing.